Docs update (#513)
segmentio/action-destinations: README.md

<img src="https://brand.segment.com/site-assets/03d0de2f/images/brand-guidelines/content/twilio/twilio-segment-color-logo-white-2x.png" style="width:50%; height=50%" alt="Twilio Segment logo" >

# Action Destinations

Action Destinations are the new way to build streaming destinations on Segment.

> Action Destinations were [launched in December 2021](https://segment.com/blog/introducing-destination-actions/) to enable customers with a customizable framework to map Segment events to their favorite 3rd party tools. This repository contains the Action Destination Definitions as well as a CLI to generate the scaffolding for new destinations and run unit tests. If you'd like to contribute, please review the [Contributing Guide](./CONTRIBUTING.md).

Reviewer: ```suggestion
Action Destinations were [launched in December 2021](https://segment.com/blog/introducing-destination-actions/) to enable customers with a customizable framework to map Segment event sources to their favorite 3rd party tools.

This repository contains the Action Destination Definitions as well as a CLI to generate the scaffolding for new destinations and run unit tests. If you'd like to contribute, please review the [Contributing Guide](./CONTRIBUTING.md).
```

> <img src="https://brand.segment.com/site-assets/03d0de2f/images/brand-guidelines/content/twilio/twilio-segment-color-logo-white-2x.png" style="width:100%; height=100%" alt="Segment logo" >

Reviewer: ```suggestion
<img src="https://brand.segment.com/site-assets/03d0de2f/images/brand-guidelines/content/twilio/twilio-segment-color-logo-white-2x.png" style="width:50%; height=50%" alt="Twilio Segment logo" >
```

# Action Destinations

> Action Destinations are a way to build streaming destinations on Segment.

Reviewer: ```suggestion
Action Destinations are the new way to build streaming destinations on Segment.
```

> Action Destinations allows builders to easily create Segment integrations with partner tools (e.g. Salesforce) by leveraging a first-class CLI to reduce boilerplate, a strong type-safety framework to enable the IDE and comiler to provide guidance and auto-complete.

Reviewer: ```suggestion
Action Destinations were [launched in December 2021](https://segment.com/blog/introducing-destination-actions/) to enable customers with a customizable framework to map Segment events to their favorite 3rd party tools. This repository contains the Action Destination Definitions as well as a CLI to generate the scaffolding for new destinations and run unit tests. If you'd like to contribute, please review the [Contributing Guide](./CONTRIBUTING.md).
```

> 
> Integrations built with Action Destinations allow users to transparently view the data they are sending to the downstream destination. Users can easily customize how data is delivered via action mappings and decide which events trigger their actions with an intuitive subscription system.

Reviewer: ```suggestion
```

- [Mapping Kit](./packages/core/src/mapping-kit/README.md)
- [Destination Kit](./packages/core/src/destination-kit/README.md)

> # Table of Contents:

Reviewer 2: ```suggestion
## Table of Contents:
```

For specific information about each CLI command, please refer to this [README](https://github.com/segmentio/action-destinations/tree/main/packages/cli).

> For instructions on how to create a new integration, see the [Create a new Destination Action](./docs/create.md) docs

Reviewer 2: ```suggestion
For instructions on how to create a new integration, see the [Create a new Destination Action](./docs/create.md) docs.
```
